.ci_support/migrations/cuda12x.yaml

```diff
@@ -66,16 +66,8 @@ __migrator:
       - 11.1                       # [(linux64 or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
       - 11.2                       # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
       - 11.8                       # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+      - 12.0                       # [(linux or win64) and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
```

Comment:
The essential bit of the re-rendering error is...

```python
  File "/opt/conda/envs/cf-feedstock-ops/lib/python3.11/site-packages/conda_smithy/variant_algebra.py", line 114, in variant_key_set_union
    return sorted(out_v, key=partial(_version_order, ordering=ordering))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/cf-feedstock-ops/lib/python3.11/site-packages/conda_smithy/variant_algebra.py", line 65, in _version_order
    return ordering.index(v)
           ^^^^^^^^^^^^^^^^^
ValueError: '12.0' is not in list
```

This is coming up because we closed out the CUDA 12.0 migrator somewhat recently ( https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/6263 )

The feedstock here still uses a vendored migrator for adding CUDA 12.x latest (currently 12.6). To keep this working, updated the `cuda_compiler_version`'s `ordering` here to include `12.0`, which fixes the re-rendering error

So we can continue to use this migrator for CUDA 12.x versions and re-render without further issues

Reply:
Ah OK, thanks for looking into this for me, John!